pkgs/os-specific/linux/rt-tests/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchurl, pkgs }:
```

Comment:
```suggestion
{ stdenv
, lib
, makeWrapper
, fetchurl
, numactl
, python3
}:
```

Reply:
Why has this been marked as resolved?

Reply:
because I have messed up and overwrote this change. fixed now